src/IncludeRelations.php

```diff
@@ -29,14 +28,14 @@ public function __construct(array $attributes = [])
     public function checkLoadableRelationsArrayIsDefined()
```

Comment:
I think we can remove these function checks, and instead initialize the `loadableRelations` and `loadableRelationsCount` to empty arrays above on the trait as attributes.

Reply:
You cannot remove because in function **loadIfLoadableRelation** used **$this->loadableRelations** and if is not defined it will thrrow error, so wee need to keep **checkLoadableRelationsArrayIsDefined** to convert to empty Array if not defined.

Reply:
You are right, now I understand.
Thank you!